src/components/TextField.tsx

```diff
@@ -14,20 +14,27 @@ const styles = (theme: Theme) =>
     },
   });
 
+  // TODO: @wangannie: make onChange not optional
```

Comment:
I  _think_ eventually I need to make `onChange` not optional, but I'd have to go through all the other existing `TextFields` and add `onChange` functions that do nothing for now.

Reply:
I agree. Don't do it in this PR. Save for a dedicated PR